app/assets/javascripts/components/authentication/reset_password_modal.es6.jsx

```diff
@@ -16,12 +16,14 @@ class ResetPasswordModal extends React.Component {
   }
 
   resetPassword() {
-    var reject = (response) => this.setState({ errors: response.error });
+    var reject = (response) => {
+      this.setState({ errors: response.message })
```

Comment:
it seems like `errors` is a single message right? maybe remove the `s` in the variable name